server/tserver/src/main/java/org/apache/accumulo/tserver/log/DfsLogger.java

```diff
@@ -186,7 +186,7 @@ public void run() {
         } catch (IOException | RuntimeException ex) {
           fail(work, ex, "synching");
         }
-        long duration = System.currentTimeMillis() - start;
+        long duration = System.nanoTime() - start;
         if (duration > slowFlushMillis) {
```

Comment:
I don't think this comparison will work correctly comparing nanoseconds to milliseconds